src/main/webapp/all_tests.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <!--
+Copyright 2009 The Closure Library Authors. All Rights Reserved.
+
+Use of this source code is governed by the Apache License, Version 2.0.
+See the COPYING file for details.
+-->
+
+  <head>
+    <title>Closure - All JsUnit Tests</title>
```

Comment:
Not sure how I feel about mixing in <style> css and <script> js directly inside an html file. Maybe write separate files for these?